client/src/js/services/FiscalService.js

```diff
@@ -33,6 +35,32 @@ function FiscalService(Api) {
       .then(service.util.unwrapHttpResponse);
   }
 
+  /**
+   * @method periodBalance
+   *
+   * @description find the balance for a specified period and fiscal year
+   * @param {object} params which contains the fiscal year id and the period number
+   * @example
+   * periodicBalance({id: 1, period_number: 0});
+   */
+  function periodicBalance(params) {
+    var url = service.url.concat(params.id, '/balance/', params.period_number);
+
+    return service.$http.get(url)
+      .then(service.util.unwrapHttpResponse);
+  }
+
+  /**
+   * @method setOpeningBalance
+   *
+   * @description set the opening balance for a fiscal year
+   */
+  function setOpeningBalance(params) {
+    var url = service.url.concat(params.id, '/opening_balance/');
+
+    return service.$http.put(url, { params: params })
```

Comment:
It creates informations if there is not any data or updates informations if data exists. We can use POST instead of PUT because the first operation is the creation of informations